modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -155,11 +154,6 @@ public String getHelp() {
     public void processOpts() {
         super.processOpts();
 
-        if (StringUtils.isEmpty(System.getenv("DART_POST_PROCESS_FILE"))) {
```

Comment:
This was already available in `AbstractDartCodegen` which this class inherits from.